.babelrc

```diff
@@ -0,0 +1,4 @@
+{
+  // "presets": ["env", "react"]
```

Comment:
Please, don't include redundant comments into the final code.

Reply:
Fixed